_layouts/training.html

```diff
@@ -0,0 +1,31 @@
+---
+pagename: Training
+---
+{% include header.html %}
+<style>
+.scaleframe { width: 650; height: 400; border: 1px solid black; }
+.scaleframe {
+  -ms-zoom: 0.5;
+  -moz-transform: scale(0.5);
+  -moz-transform-origin: 0 0;
+  -o-transform: scale(0.5);
+  -o-transform-origin: 0 0;
+  -webkit-transform: scale(0.5);
+  -webkit-transform-origin: 0 0;
+}
+</style>
+<div id="wrap">
+	<div class="container">
+		<div class="row">
+      <div class="col-md-3 hidden-xs hidden-sm">
+				<div id="toc" class="sidebar-nav-fixed pull-right" data-spy="affix" data-offset-top="60" data-offset-bottom="300">
```

Comment:
I think the sidebar here makes it look kind of bad, we only have one 'training' and it doesn't have a lot of space for rendering with the sidebar